### PR TITLE
feat: added signInController

### PR DESCRIPTION
**Pull request by v-racine**

### CODE REVIEW

File: src/controllers/createUserController.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ class CreateUserController extends BaseController {
       }
     }
 
-    // TODO: Store the id of that user inside the users cookie (in the controller layer)
+    // TODO: Store the id of that user inside the users cookie (in the controller layer)?
```
Comment (w-h-a) [inline src/controllers/createUserController.js:31]: can't we just delete this `TODO` at this point?

File: src/controllers/signInController.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      }
+    }
+
+    req.session.userId = user.id;
+
+    return res.send('You are signed in.');
+  }
+}
+
+module.exports = SignInController;
```
Comment (w-h-a) [inline src/controllers/signInController.js:32]: We could make this a little more flexible, but I s'pose it's fine for these controllers to do this. Unless we want uniformity, in which case we'd want:

```js
module.exports = { SignInController };
```

**Review by w-h-a — APPROVED** [review]
Left some comments for you to ponder. Otherwise, looks good as is! Niiiice work!

File: src/controllers/createUserController.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ class CreateUserController extends BaseController {
       }
     }
 
-    // TODO: Store the id of that user inside the users cookie (in the controller layer)
```
Comment (v-racine) [inline src/controllers/createUserController.js:31]: do we ...

Comment (v-racine) [reply src/controllers/createUserController.js:31]: rm it 🔥🔥

File: src/controllers/signInController.js (hunk 2 above)
Comment (v-racine) [reply src/controllers/signInController.js:32]: ok Mr. M, i'll do it

**Pull request merged**